src/main/menu.account.ts

```diff
@@ -0,0 +1,123 @@
+import { I18nService } from 'jslib-common/abstractions/i18n.service';
+import { MessagingService } from 'jslib-common/abstractions/messaging.service';
+
+import { isMacAppStore, isWindowsStore } from 'jslib-electron/utils';
+
+import {
+    IMenubarMenu,
+} from './menubar';
+
+import {
+    BrowserWindow,
+    dialog,
+    MenuItemConstructorOptions,
+    shell,
+} from 'electron';
```

Comment:
3rd party modules should be imported first.

Reply:
moved